website/addons/s3/templates/s3_node_settings.mako

```diff
@@ -77,3 +77,17 @@
         <p data-bind="html: message, attr.class: messageClass"></p>
     </div>
 </div>
+
+ <script>
+    // Make s3 settings variables globally accessible for relevant JS
+    <%
+        import json
+        from website.addons.s3.settings import DEFAULT_BUCKET_LOCATION
+    %>
+    window.contextVars = $.extend(true, {}, window.contextVars, {
+        s3Settings: {
+            defaultBucketLocationValue: ${json.dumps(DEFAULT_BUCKET_LOCATION['value'])},
```

Comment:
I was using sjson, n but it was causing failures. Had @abought look at it but couldn't figure out the source of the problem and decided to wait as the PR was sitting around anyways. Will switch back and see if it fails.